### PR TITLE
Cases page control 

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- [x] Add html and styling
- [x] Make it work
- [x] I18n

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
screen shot 2018-02-22 at 12 49 37

Since we have only 2 cases, this always references to the same next and prev. But there will be more cases soon.

File: source/layouts/cases_post_layout.erb [hunk 1]
```diff
@@ -100,4 +100,43 @@
...
 <%= partial "partials/social-share" %>
 
+<section class="page-controls">
+  <div class="row">
+
+    <div class="page-control-wrapper">
+
+      <%
+      prev_article = current_article.previous_article || blog.articles.first
+      next_article = current_article.next_article || blog.articles.last
```
Comment (fatboypunk) [inline source/layouts/cases_post_layout.erb]: We could also check if they're the same to the current,

Comment (sn3p) [reply source/layouts/cases_post_layout.erb]: Could skip prev article link if they are the same. Prolly need some stying adjustments.

Comment (RonaldDijkstra) [reply source/layouts/cases_post_layout.erb]: Third case is in the making! Problem #solved 👍

File: source/layouts/cases_post_layout.erb [hunk 2]
```diff
@@ -100,4 +100,43 @@
...
+              <%= t("cases.previous") %>
+          </span>
+          <h4 class="control-title">
+            <%= prev_article.title %>
+          </h4>
+        </a>
+      <% end %>
+
+      <% if next_article %>
+        <a href="<%= next_article.url %>" class="page-control page-control-next" rel="rel" title="<%= next_article.title %>">
```
Comment (sn3p) [inline source/layouts/cases_post_layout.erb]: `rel="rel"`

Comment (RonaldDijkstra) [reply source/layouts/cases_post_layout.erb]: relluh

**Pull request merged**